src/js/components/Calendar/doc.js

```diff
@@ -20,6 +20,12 @@ export const doc = Calendar => {
 
   DocumentedCalendar.propTypes = {
     ...genericProps,
+    activeRangeBound: PropTypes.oneOf(['start', 'end'])
```

Comment:
What about `activeDate`, which would always be a single date value string? This could be leveraged to indicate where keyboard navigation should start. Just thinking out loud.

Reply:
funny you say that, I had `activeDate` initially and then changed it. But maybe since that was your inclination as well, I will go back to that.